spec/models/search/bookmark_search_form_spec.rb

```diff
@@ -62,13 +62,33 @@
         end
 
         it "changes when the work is updated" do
-          work2.update_attribute(:revised_at, Time.now)
+          work2.update_attribute(:revised_at, Time.current)
           run_all_indexing_jobs
           results = BookmarkSearchForm.new(
             parent: tag, sort_column: "bookmarkable_date"
           ).bookmarkable_search_results
           expect(results.map(&:title)).to eq ["Two", "One", "Three"]
         end
+
+        it "doesn't change tied bookmarkables order on work update" do
```

Comment:
Without the tie breaker fixes, do these tests always fail or only sometimes fail?

Reply:
They failed every time I ran them.